src/components/StatusColorOptionsEditor.tsx

```diff
@@ -1,37 +1,49 @@
-import { ColorDefinition, PanelOptionsEditorProps, getColorForTheme, standardEditorsRegistry } from '@grafana/data';
-import { HorizontalGroup, Label, getTheme } from '@grafana/ui';
+import { PanelOptionsEditorProps } from '@grafana/data';
+import { ColorPicker, HorizontalGroup, Label } from '@grafana/ui';
 import { StatusPanelOptions } from 'lib/statusPanelOptionsBuilder';
+import { config } from '@grafana/runtime';
+
 import React from 'react';
 
-export const StatusColorOptionsEditor: React.FC<PanelOptionsEditorProps<StatusPanelOptions['colors']>> = ({
-  value,
-  onChange,
-}) => {
+export const StatusColorOptionsEditor: React.FC<PanelOptionsEditorProps<StatusPanelOptions['colors']>> = ({value, onChange}) => {
   // get grafana color picker and theme
-  const colorPicker = standardEditorsRegistry.get('color').editor as any;
-  const theme = getTheme();
+  // const colorPicker = standardEditorsRegistry.get('color').editor as any;
```

Comment:
Assume still wanting to keep the commented out code whilst we're testing?

Reply:
yep